### DISCUSSION TITLE
Tested on WinXP SP3 32bit - It doesn'work

### DISCUSSION THREAD

**Discussion by maa4mb** [davidrg/ckwin#429] [General]
Tested on WinXP SP3 32bit - It doesn'work because it's unable to find the entry point if-nametoindex library iphlpapi.dll.

**Comment by davidrg** [comment]
This is a known issue with the new version of the SSH backend in the current release. The SSH readme file has a section at the bottom explaining the situation along the workaround which should get it working for you.

For the next release (currently planned for January) it will be able to deal with this situation on its own by automatically loading a different build of the SSH backed in XP.

**Comment by maa4mb** [comment]
Thank you. I missed the SSH readme file, sorry.

**Reply by davidrg** [reply]
No problem! Seems I failed to mention it at all in the release notes so not surprising you missed it! I've added a note for anyone else who runs into this.
